### DISCUSSION TITLE
Custom page for default site (Settings) not working on HTTPS (custom SSL).

### DISCUSSION THREAD

**Discussion by prashantsomashekar** [NginxProxyManager/nginx-proxy-manager#2024] [Q&A]
Hi.

I set a custom page under settings which should be displayed, when it encounters a host not listed in NPM, if I understand correctly.

However, this doesn't appear to work on HTTPS (custom SSL).  The self-signed certificate (localhost) is encountered (expected) and then it throws a HTTP error after accepting the warning in the browser, to move forward.  I have not tested HTTP as I have intentionally left that closed.

Upon looking here:
https://github.com/NginxProxyManager/nginx-proxy-manager/blob/develop/backend/templates/default.conf

I don't see 443 defined.  However, HTTP (non S) should work, given that template.

Does anyone have a way to add further configuration within NPM itself to sort it out without having to modify raw files / mount a custom config in to the container?

Thank you.

**Comment by Holcane1337** [comment]
Having the same issue.
Seems there are a few issues logged that have been marked as completed but it still doesn't work.

https://github.com/npm/npm/issues/2050
https://github.com/NginxProxyManager/nginx-proxy-manager/issues/657
